src/App.vue

```diff
@@ -3,5 +3,5 @@
 </template>
 
 <script setup>
-import Example from './components/HelloWorld/Example.vue';
+import Example from './components/QHelloWorld/Example.vue';
```

Comment:
Please kill the dude who is doing that, thx <3

Reply:
<img width="1758" alt="Capture d’écran 2022-06-10 à 19 14 09" src="https://user-images.githubusercontent.com/28975415/173117765-67e6fb33-c3d1-40d0-b717-cd207fd925c5.png">

Reply:
😅